modules/nvidia-gpu-features.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * architecture/nvidia-gpu-architecture-overview.adoc
+
+
+:_content-type: CONCEPT
+[id="nvidia-gpu-features_{context}"]
+= NVIDIA GPU features for {product-title}
+
+// NVIDIA GPU Operator::
+// The NVIDIA GPU Operator is a Kubernetes Operator that enables {product-title} {VirtProductName} to expose GPUs to virtualized workloads running on {product-title}.
+// It allows users to easily provision and manage GPU-enabled virtual machines, providing them with the ability to run complex artificial intelligence/machine learning (AI/ML) workloads on the same platform as their other workloads.
+// It also provides an easy way to scale the GPU capacity of their infrastructure, allowing for rapid growth of GPU-based workloads.
+
+NVIDIA Container Toolkit::
+The NVIDIA Container Toolkit allows you to build and run GPU-accelerated containers.
+The toolkit includes a container runtime library and utilities to automatically configure
+containers to leverage NVIDIA GPUs.
+
+Node Feature Discovery (NFD) Operator::
+The Node Feature Discovery Operator (NFD) manages the detection of hardware features and configuration in an OpenShift Container Platform cluster by labeling the nodes with hardware-specific information. NFD labels the host with node-specific attributes, such as PCI cards, kernel, operating system version, and so on.
++
+The NFD Operator can be found on the Operator Hub by searching for “Node Feature Discovery”.
+
+NVIDIA AI Enterprise::
+NVIDIA AI Enterprise is an end-to-end, cloud-native suite of AI and data analytics software, optimized, certified, and supported by NVIDIA with NVIDIA-Certified Systems.
++
+NVIDIA AI Enterprise provides access to a pre-configured GPU Operator within the NVIDIA Enterprise Catalog.
+The GPU Operator is pre-configured to simplify the provisioning experience with NVIDIA AI Enterprise deployments.
```

Comment:
I'm not sure what this means. As stated above, NVIDIA AI Enterprise (NVAIE) is a suite of software (images). The images can run on OpenShift with the GPU operator, but I need to understand what is meant by "pre-configured GPU Operator"